llvm/tools/llvm-c-test/debuginfo.c

```diff
@@ -215,10 +215,26 @@ int llvm_test_dibuilder(bool NewDebugInfoFormat) {
 
   LLVMDIBuilderFinalize(DIB);
 
+  // In the new debug mode, debug records get attached to instructions.
```

Comment:
Is this code depending on the given `NewDebugInfoFormat`?

Reply:
I thought it'd be better to minimise the difference between the tests. Adding the instructions into the old-format test doesn't hurt it IMO. In a future patch I will possibly be adding some `NewDebugInfoFormat`-specific behaviour, but I think minimising the difference is still worthwhile. What do you think?